rfqmsg/messages_test.go

```diff
@@ -0,0 +1,129 @@
+package rfqmsg
+
+import (
+	"encoding/binary"
+	"testing"
+
+	"github.com/lightningnetwork/lnd/aliasmgr"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/stretchr/testify/require"
+)
+
+func intToID(scid uint64) ID {
+	var id ID
+	binary.BigEndian.PutUint64(id[24:], scid)
+	return id
+}
+
+func shortChannelIDToID(scid lnwire.ShortChannelID) ID {
+	return intToID(scid.ToUint64())
+}
+
+// TestIDSerializedScid tests that an RFQ ID can be correctly converted into a
+// custom SCID alias that lies within the valid SCID block height range.
+func TestIDSerializedScid(t *testing.T) {
+	tests := []struct {
+		id       ID
+		expected lnwire.ShortChannelID
+	}{
```

Comment:
Consider adding a `label` field to describe what the test case is trying to investigate.